specs/akhilesh/encoding_coarsener_ddp_x0.cue

```diff
@@ -0,0 +1,184 @@
+#EXP_NAME:    "encoding_coarsener"
+#EXP_VERSION: "ddp_single_gpu4_x1"
+
+#TRAINING_ROOT: "gs://tmp_2w/akhilesh/training_artifacts"
+
+//#ENCODER_CKPT:    "\(#TRAINING_ROOT)/\(#EXP_NAME)/inver_diffkeep_apply2x_x3/last.ckpt"
+//#DECODER_CKPT:    "\(#TRAINING_ROOT)/\(#EXP_NAME)/inver_diffkeep_apply2x_x3/last.ckpt"
+#ENCODER_CKPT: null
+#DECODER_CKPT: null
+
+
+//#ENC_CV: "file:///ssd/zetta_utils/test"
+#ENC_CV: "gs://fafb_v15_aligned/v0/img/img"
+//#ENC_CV: "gs://tmp_2w/testvol/"
+//#ENC_CV: "gs://fafb_v15_aligned/v0/experiments/emb_fp32/baseline_downs_emb_m2_m5_x0"
+
+
+///////////////////////////////////////////////////////////////////
+//////////////////////// Training Spec ////////////////////////////
+///////////////////////////////////////////////////////////////////
+
+"@type": "lightning_train"
+regime: {
+	"@type": "EncodingCoarsenerRegime"
+	lr:      4e-4
+	encoder: {
+		"@type":   "load_weights_file"
+		model:     #ENCODER_MODEL
+		ckpt_path: #ENCODER_CKPT
+		component_names: ["encoder"]
+	}
+	decoder: {
+		"@type":   "load_weights_file"
+		model:     #DECODER_MODEL
+		ckpt_path: #DECODER_CKPT
+		component_names: ["decoder"]
+	}
+	apply_counts: [1, 2, 3]
+	invar_angle_range: [1, 180]
+	invar_mse_weight: 0.1
+	diffkeep_angle_range: [1, 5]
+	diffkeep_weight: 0.1
+}
+trainer: {
+	"@type":            "ZettaDefaultTrainer"
+	accelerator:        "cuda"
+	devices:            4
+	strategy:           {
+		"@type":					"pl.DDPStrategy"
```

Comment:
Should we set the default strategy to this so that the user doesn't have to worry? Sounds like we won't need another strategy as of now